indexer/packages/postgres/src/db/migrations/migration_files/20240913142157_change_affiliate_info_decimal_precision_and_add_total_referred_volume.ts

```diff
@@ -1,21 +1,22 @@
 import * as Knex from 'knex';
 
+// No data has been stored added at time of commit
 export async function up(knex: Knex): Promise<void> {
   return knex.schema.alterTable('affiliate_info', (table) => {
     // null indicates variable precision whereas not specifying will result in 8,2 precision,scale
-    table.decimal('affiliateEarnings', null).alter();
-    table.decimal('totalReferredFees', null).alter();
-    table.decimal('referredNetProtocolEarnings', null).alter();
+    table.decimal('affiliateEarnings', null).notNullable().defaultTo(0).alter();
+    table.decimal('totalReferredFees', null).notNullable().defaultTo(0).alter();
+    table.decimal('referredNetProtocolEarnings', null).notNullable().defaultTo(0).alter();
 
-    table.decimal('referredTotalVolume', null).notNullable();
+    table.decimal('referredTotalVolume', null).notNullable().defaultTo(0);
   });
 }
 
 export async function down(knex: Knex): Promise<void> {
   return knex.schema.alterTable('affiliate_info', (table) => {
-    table.decimal('affiliateEarnings').alter();
-    table.decimal('totalReferredFees').alter();
-    table.decimal('referredNetProtocolEarnings').alter();
+    table.decimal('affiliateEarnings').notNullable().defaultTo(0).alter();
+    table.decimal('totalReferredFees').notNullable().defaultTo(0).alter();
+    table.decimal('referredNetProtocolEarnings').notNullable().defaultTo(0).alter();
```

Comment:
Lets try not to make this a habit, it's better to create a new migration to delete the rows above and change them to notNullable and default to 0. Especially since the cutoff for the release is today

Reply:
Ok let me just make a new migration this time around as well then